lib/actions/purgeTable.js

```diff
@@ -7,9 +7,30 @@ const findPrimaryKey = context => {
 
   return context.dynamodb.describeTable(params).promise()
     .then(tableDescription => {
-      context.primaryKey = tableDescription.Table.KeySchema.find(
-        element => element.KeyType === 'HASH').AttributeName
-      return context
+      const keys = {};
+
+      let primaryHash = null;
+      let primaryRange = null;
+
+      try {
+        if(primaryHash = tableDescription.Table.KeySchema.find(element => element.KeyType === 'HASH').AttributeName);
+        if(primaryRange = tableDescription.Table.KeySchema.find(element => element.KeyType === 'RANGE').AttributeName);
+      } catch (err) {
+        // it's ok, if entry this mean that one not exists.
+      }
+
+      if (primaryHash !== null) {
+        keys.hash = primaryHash;
+      }
+
+      if (primaryRange !== null) {
+        keys.range = primaryRange;
+      }
+
+      context.primaryKey = keys;
+
+      return context;
+      
```

Comment:
There is a lot of style issues in those changes.
Please run `npm run lint` and fix all reported issues.